libraries/botbuilder-dialogs/src/dialogManager.ts

```diff
@@ -51,64 +53,74 @@ export interface DialogManagerConfiguration {
 }
 
 export class DialogManager extends Configurable {
-    private dialogSet: DialogSet = new DialogSet();
-    private rootDialogId: string;
-    private dialogStateProperty: string;
+    private _rootDialogId: string;
+    private readonly _dialogStateProperty: string;
```

Comment:
These changes are unnecessary, we don't _normally_ follow this naming convention in JS... But since _initialTurnState already has the `_`, let's just leave it in so that the class is consistent.